community/runc/APKBUILD

```diff
@@ -21,8 +20,8 @@ source="runc-$_commit.tar.gz::https://github.com/opencontainers/runc/archive/$_c
 builddir="$srcdir/src/github.com/opencontainers/runc"
 
 # secfixes:
-#   1.0.0_rc6-r1:
-#     - CVE-2019-5736
+#   1.0.0_rc6-r2:
+#     - CVE-2019-5736 (fix improved vs. 1.0.0_rc6-r1)
```

Comment:
the secfixes are read by scripts. Im not sure it is a good idea to add the extra comment there